server/models/users.js

```diff
@@ -0,0 +1,77 @@
+const getUserModel = (sequelize, Sequelize) => {
+  const flags = {
+    timestamps: false,
+    freezeTableName: true
+  };
+  const userSchema = {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      unique: true,
+      allowNull: false
+    },
+    firstName: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    middleName: {
+      type: Sequelize.STRING
+    },
+    lastName: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true
+    },
+    gender: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    biography: {
+      type: Sequelize.TEXT
+    },
+    mobileNumber: {
+      type: Sequelize.CHAR
```

Comment:
This should be of type string. Also, remember we have a database design already, stick to the attribute/field names on it.

Reply:
This follows the design of that was reviewed by the team, please kindly confirm from your design schema